dhcpv4/nclient4/lease.go

```diff
@@ -0,0 +1,227 @@
+//This is lease managment for nclient4
+
+package nclient4
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/vishvananda/netlink"
```

Comment:
We moved away from `vishvananda/netlink`, check #271. Can you use `jsimonetti/rtnetlink`?

Reply:
ok, will take a look